ext/pcre/config0.m4

```diff
@@ -67,7 +67,7 @@ else
   pcre2lib/pcre2_tables.c pcre2lib/pcre2_ucd.c pcre2lib/pcre2_valid_utf.c pcre2lib/pcre2_xclass.c \
   pcre2lib/pcre2_find_bracket.c pcre2lib/pcre2_convert.c pcre2lib/pcre2_extuni.c pcre2lib/pcre2_script_run.c"
   PHP_PCRE_CFLAGS="-DHAVE_CONFIG_H -I@ext_srcdir@/pcre2lib -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1"
-  PHP_NEW_EXTENSION(pcre, $pcrelib_sources php_pcre.c, no,,$PHP_PCRE_CFLAGS)
+  PHP_NEW_EXTENSION(pcre, $pcrelib_sources php_pcre.c, no,,\\$(PHP_PCRE_CFLAGS))
```

Comment:
Are you going through subst because the check is done after the NEW_EXTENSION call? Can't we do the check first?

Reply:
I think that would also be possible.

@remicollet, could you please have a look. I'm pretty lost with autoconf. :)

Reply:
Okay, just pushed a commit which changes the order.